tensorflow-examples/src/main/java/org/tensorflow/model/examples/objectdetection/FasterRcnnInception.java

```diff
@@ -0,0 +1,302 @@
+
+            ReadFile readFile = tf.io.readFile(fileName);
+
+            Session.Runner runner = s.runner();
+            s.run(tf.init());
+
+            DecodeJpeg.Options options = DecodeJpeg.channels(3L);
+            DecodeJpeg decodeImage = tf.image.decodeJpeg(readFile.contents(), options);
+            //fetch image from file
+            TUint8 outputImage = (TUint8) runner.fetch(decodeImage).run().get(0);
+            Shape imageShape = outputImage.shape();
+            long[] shapeArray = imageShape.asArray();
+
+            //The given SavedModel SignatureDef input
+            Map<String, Tensor> feed_dict = new HashMap<>();
+            feed_dict.put("input_tensor", reshapeTensor(outputImage));
+            //The given SavedModel MetaGraphDef key
+            Map<String, Tensor> outputTensorMap = model.function("serving_default").call(feed_dict);
+
+            //detection_classes is a model output name
+            TFloat32 detectionClasses = (TFloat32) outputTensorMap.get("detection_classes");
+            TFloat32 detectionBoxes = (TFloat32) outputTensorMap.get("detection_boxes");
+            TFloat32 numDetections = (TFloat32) outputTensorMap.get("num_detections");
+            TFloat32 detectionScores = (TFloat32) outputTensorMap.get("detection_scores");
+
+            int numDetects = (int) numDetections.getFloat(0);
+            if (numDetects > 0) {
+                try {
+                    BufferedImage bufferedImage = ImageIO.read(new File(imagePath));
+                    Graphics2D graphics2D = bufferedImage.createGraphics();
+
+                    //TODO tf.image.combinedNonMaxSuppression
+                    for (int n = 0; n < numDetects; n++) {
+                        //put probability and position in outputMap
+                        if(detectionScores.getFloat(0,n)> 0.3f) {
+                            Float classVal = detectionClasses.getFloat(0, n);
+                            //TODO tf.image.drawBoundingBoxes
+                            int x1 = (int) (shapeArray[1] * detectionBoxes.getFloat(0, n, 1));
+                            int y1 = (int) (shapeArray[0] * detectionBoxes.getFloat(0, n, 0));
+                            int x2 = (int) (shapeArray[1] * detectionBoxes.getFloat(0, n, 3));
+                            int y2 = (int) (shapeArray[0] * detectionBoxes.getFloat(0, n, 2));
+                            graphics2D.setPaint(Color.RED);
+                            graphics2D.setStroke(new BasicStroke(5));
+                            graphics2D.drawRect(x1, y1, x2 - x1, y2 - y1);
+                            graphics2D.setPaint(Color.BLACK);
+                            graphics2D.drawString(cocoTreeMap.get(classVal - 1) + " " + classVal, x1, y1);
+                        }
+                    }
+                    //TODO tf.image.encodeJpeg
+                    ImageIO.write(bufferedImage, "jpg", new File("image2rcnn.jpg"));
+                } catch (IOException e) {
```

Comment:
Please log something at least if it throws IOException, otherwise this could produce no output and be confusing for people who run it.

Reply:
Will do